contents/tutorials/performance-metrics.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Understanding page load time metrics in performance monitoring
+sidebar: Docs
+showTitle: true
+author: ['lior-neu-ner']
+date: 2023-04-12
+featuredImage: todo
+tags: []
+---
+
+Waiting for slow websites is like watching paint dry. It's the bane of productivity, the destroyer of efficiency, and the enemy of progress.
+
+In this tutorial, we'll show you how to understand different metrics on how quickly your pages load and how to identify opportunities for improvement using Posthog's [performance monitoring in session recordings tool](https://posthog.com/blog/posthog-changelog#performance-monitoring-in-session-recordings).
+
+## Setup
+
+First, ensure that session recordings are [enabled]((https://posthog.com/docs/session-replay/manual)). Next, navigate to the Recordings tab on your Posthog Dashboard. Finally, click on the Network tab in the sidebar next to or below the video.
+
+![](https://i.imgur.com/nr0QSSy.png)
+
+Here you'll see all the network requests that are made during the page load and user session, as well as how long they took. You'll also see the time taken for First Contentful Paint, DOM interactive, and Page Loaded. 
+
+![](https://i.imgur.com/Qlqt4c0.png)
+
+We'll go through what each one of these three metrics mean and how to improve them.
+
+## First Contentful Paint
+[First Contentful Paint](https://developer.mozilla.org/en-US/docs/Glossary/First_contentful_paint) (FCP) measures the time from when the page starts loading to when any element of the page's content is rendered on the screen. Note that it does not measure the time it takes entire page to load.
+
+[![FCP happens in the second frame, as that's when the first text and image elements are rendered to the screen.](https://web-dev.imgix.net/image/admin/3UhlOxRc0j8Vc4DGd4dt.png?auto=format&w=1600)](https://web.dev/fcp/)
+*FCP happens in the second frame, as that's when the first text and image elements are rendered to the screen. [Source](https://web.dev/fcp)* 
+
+
+[According to Chrome team](https://web.dev/fcp/), a good FCP time is 1.8 seconds while anything above 3 seconds is considered poor. Factors that can affect FCP are:
```

Comment:
```suggestion
[According to the Chrome team](https://web.dev/fcp/), a good FCP time is 1.8 seconds while anything above 3 seconds is considered poor. Factors that can affect FCP are:
```

Reply:
Idea for making this section/post more fun - what if you embedded a timer in where people could actually feel how long 1.8s or 3s etc. actually took?

Alternatively/additionally - can we share some anecdote about how each X second to page load reduces your chance of converting someone by Y? (or similar, doesn't have to be that exact anecdote!)

Reply:
GIF of 1.8 vs 3s page load might be easier?

Reply:
Interesting idea! I'll play around with a few things here

Reply:
I played around with this a bit but didn't think it added much value (and I feel that anything that doesn't add value detracts value :) )